posts/2023-02-15-hackaton-made-by-cesi.adoc

```diff
@@ -0,0 +1,40 @@
+
+Participants had the chance to attend three exciting conferences on the topics of Low Tech, Women in Engineering and the Engineer of the Future during the morning. The objective was to encourage participants to integrate or build on these topics in the research and development of their ideas. The conferences brought a new perspective to the participants on the current and future challenges of engineering. Low Tech was presented as a creative and sustainable alternative to high-tech technologies for dealing with emergency situations. The place of women in engineering was discussed, highlighting the importance of diversity in engineering in order not to deprive ourselves of any skills and to build an inclusive as well as equitable solution. The engineer of the future was presented as being more ethical, sustainable and aware of the consequences of their actions.
+
+image::../media/2023-02-15-hackaton-made-by-cesi/conf.jpeg[Conference,640,480]
+
+At noon, participants were fed by a caterer who prepared a variety of seasonal burgers. They were tasty and of good quality and the fries were homemade. The ingredients used were local and produced in an environmentally friendly manner, supporting small producers and promoting sustainable food. Everyone enjoyed the lunch break, which allowed them to relax and recharge for the rest of the intense day of thinking and working. It reinforced their motivation to continue working on their ideas and innovative projects.
+
+Lunch was a key moment of the hackathon, as participants were able to discuss and exchange ideas with their teammates. This strengthened their cohesion and collaboration, which contributed to the success of their projects.
+
+image::../media/2023-02-15-hackaton-made-by-cesi/caterer.jpeg[Caterer,640,480]
+
+The teams then went back to work to develop their ideas and transform them into a real innovative concept. The afternoon was also dedicated to the preparation of the final presentation during which the different teams had to face a jury in charge of establishing the ranking and an audience composed of all the coaches and parents.
+
+image::../media/2023-02-15-hackaton-made-by-cesi/public.jpeg[Public,640,480]
+
+The first place team conceptualized a survival school to teach people the skills needed to cope with an apocalyptic situation. The second place team developed the idea of a virtual reality simulation to help people prepare mentally and physically for the apocalypse. And finally the team completing the podium made a self-sufficient dome system, which can provide food, water and energy for people in an emergency situation.
+The day ended with a prize-giving ceremony for each team. The first team received airpods, the second team received wireless speakers, the third team received gift cards and the fourth and fifth team received chocolate sets made by a local company.
```

Comment:
```suggestion
The day ended with a prize-giving ceremony. The first team was awarded airpods, the second team received wireless speakers, the third team was given gift cards while the fourth and fifth team received chocolate sets made by a local company.
```

This is not a strict recommendation. Just giving ideas on verbs to use instead of repeating "received". :)